data.sql

```diff
@@ -2,4 +2,15 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
 VALUES ('Agumon', '2020-02-03', 0, 't', 10.23), 
 	('Gabumon', '2018-11-15', 2, 't', 8.00),
 	('Pikachu', '2021-01-07', 1, 'f', 15.04),
-	('Devimon', '2017-05-12', 5, 't', 11.00);
\ No newline at end of file
+	('Devimon', '2017-05-12', 5, 't', 11.00);
+-- -----------------------------------------------------------------------------------------------------
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) 
+VALUES ('Charmander', '2020-02-08', 0, 'f', -11.00), 
+	('Plantmon', '2021-11-15', 2, 't', -5.70),
+	('Squirtle', '1993-04-02', 3, 'f', -12.13),
+	('Angemon', '2005-06-12', 1, 't', -45.00),
+	('Boarmon', '2005-06-07', 7, 't', 20.00),
```

Comment:
- [ ] It would be better if we change the weight from `20.00` we have currently to `20.4` as we have in the requirements
![Screenshot from 2023-07-11 09-28-53](https://github.com/Theodoraldo/vet-clinic-db/assets/46388113/f27a85c6-eb58-4837-99c1-95af9840dbdf)